editor/blocks/heading-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, prop } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/heading', {
+	title: 'Heading',
+	icon: 'heading',
+
+	attributes: {
+		value: query( 'h1,h2,h3,h4,h5,h6', html() ),
```

Comment:
Since the other matcher functions accept a selector, you can leave out the `query`:

```js
value: html( 'h1,h2,h3,h4,h5,h6' )
```

The `query` comes in handy particularly when needing to retrieve an array of values, like in a quote which contains many paragraphs whose text should be assigned as `text: [ '...', '...' ]`.